### PR TITLE
[test] Fix energy reference value and difference of QE small

### PR DESCRIPTION
**Pull request by lucamar**
The fix applies to the modulefile `QuantumESPRESSO/6.6a2-CrayPGI-20.10-cuda`: the total energy of the small test case changed with the new Cray Programming Environment 20.10 and the new cudatoolkit 11.0.

### CODE REVIEW

**Comment by lucamar** [comment]
I have temporarily increased the energy difference allowed by the sanity check, otherwise the test would fail either on Dom or Piz Daint until we have different default Cray PE and CUDA version on the two systems, since the total energy difference is ~ `8e-08`: 
https://jenkins.cscs.ch/blue/organizations/jenkins/ReframeCI/detail/ReframeCI/4450/pipeline

**Review by teojgo — CHANGES_REQUESTED** [review]
Just allign the FIXME with the following line

**Review by teojgo — APPROVED** [review]

**Comment by teojgo** [comment]
@jenkins-cscs retry all

File: cscs-checks/apps/quantumespresso/quantumespresso_check.py [hunk 1]
```diff
@@ -57,7 +56,9 @@ def __init__(self, scale, variant):
         energy_diff = sn.abs(energy-energy_reference)
         self.sanity_patterns = sn.all([
             self.sanity_patterns,
-            sn.assert_lt(energy_diff, 1e-8)
+            # FIXME temporarily increase energy difference
```
Comment (teojgo) [inline cscs-checks/apps/quantumespresso/quantumespresso_check.py:59]:
```suggestion
            # FIXME temporarily increase energy difference
```

**Review by teojgo — CHANGES_REQUESTED** [review]

**Review by teojgo — APPROVED** [review]

**Comment by lucamar** [comment]
The test `unittests/test_utility.py` failed on Eiger, however there is `Lmod` on the system:
```
unittests/test_utility.py:270: AssertionError
[...]
SKIPPED [2] unittests/test_utility.py:1406: could not find a sane TMod installation: [Errno 2] No such file or directory: 'modulecmd': 'modulecmd'
SKIPPED [2] unittests/test_utility.py:1406: could not find a sane TMod4 installation: [Errno 2] No such file or directory: 'modulecmd': 'modulecmd'
FAILED unittests/test_utility.py::test_git_repo_exists - AssertionError: asse...
```

**Comment by vkarak** [comment]
@jenkins-cscs retry eiger

**Review by vkarak — APPROVED** [review]
lgtm

**Pull request merged**